### DISCUSSION TITLE
Help setting up cloud-janitor for brew?

### DISCUSSION THREAD

**Discussion by faermanj** [jreleaser/jreleaser#1226] [Q&A]
Hello fellows,

Would anyone be interested in helping our project release with brew?
I managed to setup jreleaser basics but failed to get brew working.

Also, is there any course, consulting, workshop or other resources for getting help? Might be interesting to build...

Here is our repo:
https://github.com/CaravanaCloud/cloud-janitor

Thank you!

**Comment by aalmiray** [comment]
Using Jlink with Quarkus can get messy https://github.com/jreleaser/jreleaser/issues/1055
Have you had a look at https://github.com/jreleaser/helloworld-java-jlink ? Adding brew support should be as simple as defining

```yml
packagers:
  brew:
    active: ALWAYS
    multiPlatform: true
```
